src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -114,26 +116,38 @@ ClosureRemap build_remap(const Function& fbody, const DCOFFParams& params_to) {
 }
 
 void apply_remap(Subgraph& fcall, const ClosureRemap& m) {
-    std::vector<ov::Tensor> new_closure;
+    std::vector<ov::npuw::weights::LazyTensor> new_transformations;
```

Comment:
well, DCOFF shouldn't be part of this change too.
Need to discuss how we've got into this.

Reply:
It should work fine - I tested it